--- origsrc/qmmp-0.7.4/src/app/CMakeLists.txt	2014-02-12 04:32:15.996106800 -0600
+++ src/qmmp-0.7.4/src/app/CMakeLists.txt	2014-02-12 04:33:21.731866700 -0600
@@ -14,7 +14,6 @@ INCLUDE(FindQt4)
 
 include(${QT_USE_FILE})
 
-ADD_DEFINITIONS( -Wall )
 ADD_DEFINITIONS(-DQT_NO_DEBUG)
 ADD_DEFINITIONS(-DQT_THREAD)
 
@@ -35,6 +34,9 @@ SET(app_SRCS
     lxdesupport.cpp
     qmmpapplication.cpp
 )
+if(WIN32)
+SET(app_SRCS ${app_SRCS} images/56x56/icon.rc)
+endif(WIN32)
 
 SET(app_HDRS lxdesupport.h)
 
@@ -58,6 +60,7 @@ include_directories(${CMAKE_CURRENT_BINA
 ADD_EXECUTABLE(qmmp ${app_SRCS} ${app_MOC_SRCS} ${app_RCC_SRCS} ${app_HDRS})
 target_link_libraries(qmmp ${QT_LIBRARIES} libqmmp qmmpui)
 add_dependencies(qmmp qmmpui libqmmp)
+set_target_properties(qmmp PROPERTIES LINK_FLAGS "${CMAKE_CREATE_WIN32_EXE}")
 install(TARGETS qmmp DESTINATION bin)
 install(FILES qmmp.desktop DESTINATION share/applications)
 install(FILES images/16x16/qmmp.png DESTINATION share/icons/hicolor/16x16/apps)
